packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -237,6 +242,47 @@ export class InteractionTracker {
     stopWatch.stopAndRecord();
     delete this.interactions[interactionId];
   }
+
+  // Monkey-patch `History API` to detect route transitions.
+  // This is necessary because there might be some cases when
+  // there are several interactions being tracked at the same time
+  // but if there is an user interaction that triggers a route transition
+  // while those interactions are still in tracking, only that interaction
+  // will have a `Navigation` name. Otherwise, if this is not patched, the
+  // other interactions will change the name to `Navigation` even if they
+  // did not cause the route transition.
+  private patchHistoryApi() {
+    const pushState = history.pushState;
+    history.pushState = (
+      //tslint:disable:no-any
+      data: any,
```

Comment:
Can you use `unknown` here instead of `any` and so remove the `tslint` comment? Same below as well.